data/mods/gen8/moves.ts

```diff
@@ -211,6 +216,10 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		inherit: true,
 		isNonstandard: null,
 	},
+	futuresight: {
+		inherit: true,
+		flags: {futuremove: 1},
+	},
```

Comment:
Is this a deliberate behaviour change? What about Doom Desire?

Reply:
Future Sight has the allyanim flag in 9 but not in 8.